src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/OrderStatus.sql

```diff
@@ -1,7 +1,7 @@
 ﻿CREATE TABLE dbo.OrderStatus
 (
-    OrderStatusId INT IDENTITY(1,1), 
-    [Name] NVARCHAR(30) NULL,
+    OrderStatusId INT NOT NULL,

```

Comment:
What's the reason for changing this from being an `IDENTITY` column?

Reply:
I felt like this ensures more control over the ID of each status as these values are represented in the Domain layer, wouldn't want them to get out of step e.g. change in the order of insert. Also makes it clear what the ID is & I didn't have to run the select to find out.

Reply:
Admittedly, the `IDENTITY` here probably didn't really add much as these values will only ever be changed/added to as part of a release.